File: back-end/netcore/VstepPractice.API/Models/Entities/QuestionOption.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace VstepPractice.API.Models.Entities;

public class QuestionOption : BaseEntity
{
    [Column("questionId")] 
    public int QuestionId { get; set; }
    [Column("content")] 
    public string? Content { get; set; } = "";

    [Column("isCorrect")] 
    public bool IsCorrect { get; set; } = false;
    [Column("orderNum")]
    public int OrderNum { get; set; }
    // Navigation property
    [ForeignKey(nameof(QuestionId))] 
    public virtual Question Question { get; set; } = default!;
}

File: back-end/db/models/question-option.js
'use strict'
const { Model } = require('sequelize')
module.exports = (sequelize, DataTypes) => {
  class QuestionOption extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
      QuestionOption.belongsTo(models.Question, {
        foreignKey: 'questionId',
        as: 'question',
      })
    }
  }

  QuestionOption.init(
    {
      content: DataTypes.TEXT,
      isCorrect: {
        type: DataTypes.BOOLEAN,
        allowNull: false,
        defaultValue: false,
      },
      orderNum: DataTypes.INTEGER,
    },
    {
      sequelize,
      modelName: 'QuestionOption',
    },
  )
  return QuestionOption
}
